.pre-commit-config.yaml

```diff
@@ -5,21 +5,8 @@
 # Update hooks:
 #   $ pre-commit autoupdate
 ---
-exclude: |
-  (?x)(
-    (^vendor/)|
-    (.deepcopy.go$)
-  )
+exclude: ".*.deepcopy.go"
```

Comment:
Think we need to escape? But since most of our validators use `pass_filenames: false` this is ignored.

`.*\.deepcopy\.go$`

Reply:
".*.deepcopy.go" works :) I tested it out manually locally by editing some deepcopy.go in dirty manner and committing them and pre-commit was ignoring them.

Same thing I recently confirmed and committed here as well - https://github.com/openshift/reference-addon/blob/main/.pre-commit-config.yaml#L1

Though, if you noticed some edge-case/issue with the above regex do let me know. I won't be surprised if I missed one as regexes are edgy too deal with 😂

Reply:
perfect.